portfolio/src/main/webapp/blog.html

```diff
@@ -45,6 +45,29 @@ <h4>date written</h4>
       </div> 
     </div>
 
+    <div id="comments">
+      <h3>Submit a comment!</h3>
+      <form action="/data" method="POST">
+
```

Comment:
nit: i think we can get rid of the spacing since it doenst make a difference in the actual HTML